servicetalk-concurrent-internal/src/main/java/io/servicetalk/concurrent/internal/ThrowableUtil.java

```diff
@@ -58,4 +58,13 @@ public static boolean matches(Throwable original, Class<? extends Throwable> toM
         }
         return false;
     }
+
+    /**
+     * Gets not null cause of original exception or the original exception.
+     * @param original {@link Throwable} original exception.
+     * @return {@link Throwable#getCause()} if it is not null, or original
+     */
+    public static Throwable getCause(Throwable original) {
```

Comment:
Because it's not clear that this method is useful for some other places I would recommend to keep it as a `private static` method in `BlockingUtils` with the name `executionExceptionCause(ExecutionException)`. No need to add javadoc in this case.
P.S. Try to keep one empty line before `@param` in future PRs.

Reply:
Done. Thanks for the review